assets/Scripts/Managers/GameManager.cs

```diff
@@ -516,6 +535,7 @@ private static void MakeLogEntry(string _hitType)
             outsetHit,
             backtracking,
             errorTargetID,
+            temp,
```

Comment:
temp might not be the best name for the variable

Reply:
@bastianilso  I forgot to change it when I was testing 😄